packages/kit/src/core/dev/index.js

```diff
@@ -200,43 +200,95 @@ class Watcher extends EventEmitter {
 
 		/** @type {import('types/internal').SSRManifest} */
 		this.manifest = {
-			assets: manifest_data.assets,
-			layout: manifest_data.layout,
-			error: manifest_data.error,
-			routes: manifest_data.routes.map((route) => {
-				if (route.type === 'page') {
+			appDir: this.config.kit.appDir,
+			assets: new Set(manifest_data.assets.map((asset) => asset.file)),
+			_: {
```

Comment:
`_` is the property name or is it just a special syntax I'm unfamiliar with? Assuming it's the name, can we come up with something more descriptive?

Reply:
the idea is that the manifest is considered public API so that adapters can use stuff like `assets` and `appDir` (which the cloudflare adapter uses to handle requests for static assets), but there's also a private area where we make no guarantees of stability. i figured `_` is as good a name as any since it's a commonly used convention for 'private stuff'